bin/1.18/behat-prepare-simplesaml1.18.0.sh

```diff
@@ -46,118 +50,102 @@ git clone -b "$TERMINUS_ENV" "$PANTHEON_GIT_URL" "$PREPARE_DIR"
-# Silence output so as not to show the password.
+# Append to existing config as you had it
+echo "// This variable was added by behat-prepare.sh." >> "$PREPARE_DIR/private/simplesamlphp/config/authsources.php"
 {
-  echo "\$wordpress_admin_password = '${WORDPRESS_ADMIN_PASSWORD}';" >> "$PREPARE_DIR"/private/simplesamlphp/config/authsources.php
-} &> /dev/null
-echo "\$wordpress_admin_username = '${WORDPRESS_ADMIN_USERNAME}';" >> "$PREPARE_DIR"/private/simplesamlphp/config/authsources.php
-echo "\$wordpress_admin_email = '${WORDPRESS_ADMIN_EMAIL}';" >> "$PREPARE_DIR"/private/simplesamlphp/config/authsources.php
-cat "$BASH_DIR"/authsources.php.additions >> "$PREPARE_DIR"/private/simplesamlphp/config/authsources.php
-cat "$BASH_DIR"/config.php.additions >> "$PREPARE_DIR"/private/simplesamlphp/config/config.php
-
-# Copy identify provider configuration files into their appropriate locations
-cp "$BASH_DIR"/saml20-idp-hosted.php  "$PREPARE_DIR"/private/simplesamlphp/metadata/saml20-idp-hosted.php
-cp "$FIXTURES_DIR"/shib13-idp-hosted.php  "$PREPARE_DIR"/private/simplesamlphp/metadata/shib13-idp-hosted.php
-cp "$BASH_DIR"/saml20-sp-remote.php  "$PREPARE_DIR"/private/simplesamlphp/metadata/saml20-sp-remote.php
-cp "$FIXTURES_DIR"/shib13-sp-remote.php  "$PREPARE_DIR"/private/simplesamlphp/metadata/shib13-sp-remote.php
-
-# Enable the exampleauth module
-touch "$PREPARE_DIR"/private/simplesamlphp/modules/exampleauth/enable
-
-# Generate a certificate SimpleSAMLphp uses for encryption
-# Because these files are in ~/code/private, they're inaccessible from the web
-openssl req -newkey rsa:2048 -new -x509 -days 3652 -nodes -out "$PREPARE_DIR"/private/simplesamlphp/cert/saml.crt -keyout "$PREPARE_DIR"/private/simplesamlphp/cert/saml.pem -batch
-
-# Modify the login template so Behat can submit the form
-sed -i  -- "s/<button/<button id='submit'/g" "$PREPARE_DIR"/private/simplesamlphp/modules/core/templates/loginuserpass.tpl.php
-sed -i  -- "s/this.disabled=true; this.form.submit(); return true;//g" "$PREPARE_DIR"/private/simplesamlphp/modules/core/templates/loginuserpass.tpl.php
-# Second button instance shouldn't have an id
-sed -i  -- "s/<button id='submit' class=\"btn\" tabindex=\"6\"/<button class=\"btn\" tabindex=\"6\"/g" "$PREPARE_DIR"/private/simplesamlphp/modules/core/templates/loginuserpass.tpl.php
+  echo "\$wordpress_admin_password = '${WORDPRESS_ADMIN_PASSWORD}';"
+} &> /dev/null >> "$PREPARE_DIR/private/simplesamlphp/config/authsources.php"
+echo "\$wordpress_admin_username = '${WORDPRESS_ADMIN_USERNAME}';" >> "$PREPARE_DIR/private/simplesamlphp/config/authsources.php"
+echo "\$wordpress_admin_email = '${WORDPRESS_ADMIN_EMAIL}';" >> "$PREPARE_DIR/private/simplesamlphp/config/authsources.php"
+cat "$BASH_DIR/authsources.php.additions" >> "$PREPARE_DIR/private/simplesamlphp/config/authsources.php"
+cat "$BASH_DIR/config.php.additions"     >> "$PREPARE_DIR/private/simplesamlphp/config/config.php"
```

Comment:
remove extraneous whitespace
```suggestion
cat "$BASH_DIR/config.php.additions" >> "$PREPARE_DIR/private/simplesamlphp/config/config.php"
```